core/common/src/main/resources/com/alipay/sofa/rpc/common/rpc-config-default.json

```diff
@@ -258,6 +258,8 @@ PS:大家也看到了，本JSON文档是支持注释的，而标准JSON是不支
   "compress.open": false,
   // 开启压缩的大小基线
   "compress.size.baseline": 2048,
+  //Whether the Http2 Cleartext protocol client uses Prior Knowledge to start Http2
+  "transport.h2c.use_prior_knowledge": false,
```

Comment:
default value can set to `true`.

Reply:
It 's ok to set `true` because we know exactly the protocol of provider is `h2c`.